doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -296,6 +296,22 @@ Trusted Firmware-M
 * Updated :file:`tfm_platform_system.c` to fix a bug that returned ``TFM_PLATFORM_ERR_SUCCESS`` instead of ``TFM_PLATFORM_ERR_INVALID_PARAM`` when the address passed is outside of the accepted read range.
 * Added a test case for the secure read service that verifies that only addresses within the accepted range can be read.
 
+Crypto
+------
+
+* hw_unique_key: Refactor the HUK_HAS_* macros to be defined/undefined instead of 1/0.
```

Comment:
```suggestion
* hw_unique_key: Refactored the HUK_HAS_* macros to be defined/undefined instead of 1/0.
```
Also, is hw_unique_key supposed to be a reference?